data_steward/tools/create_tier.py

```diff
@@ -254,16 +256,29 @@ def parse_deid_args(args=None):
                         action='store_true',
                         required=False,
                         help='Log to the console as well as to a file.')
+    parser.add_argument(
+        '-qa',
+        '--qa_handoff_date',
+        dest='qa_handoff_date',
+        help='Will default to today\'s date for QA dataset handoff',
+        default=datetime.today(),
+        required=False)
     return parser.parse_args(args)
 
 
 def main(raw_args=None):
     args = parse_deid_args(raw_args)
     pipeline_logging.configure(level=logging.DEBUG,
                                add_console_handler=args.console_log)
-    create_tier(args.credentials_filepath, args.project_id, args.tier,
-                args.idataset, args.release_tag, args.deid_stage,
-                args.target_principal)
+
+    if args.deid_stage == BASE:
+        create_tier(args.credentials_filepath, args.project_id, args.tier,
+                    args.idataset, args.release_tag, args.deid_stage,
+                    args.target_principal, args.qa_handoff_date)
+    else:
+        create_tier(args.credentials_filepath, args.project_id, args.tier,
+                    args.idataset, args.release_tag, args.deid_stage,
+                    args.target_principal)
```

Comment:
There is no need to change lines 264-266.  You can just get the date in `create_tier()` if the datastage contains `base` in it's string name.  No need for this conditional.